CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Support scope attributes and make them as identifying for `Meter` in `go.opentelemetry.io/otel` and `go.opentelemetry.io/otel/sdk/metric`. (#5926)
 - Support scope attributes and make them as identifying for `Logger` in `go.opentelemetry.io/otel` and `go.opentelemetry.io/otel/sdk/log`. (#5925)
 - Make schema URL and scope attributes as identifying for `Tracer` in `go.opentelemetry.io/otel/bridge/opentracing`. (#5931)
+- Add int64Measures and float64Measures to `go.opentelemetry.io/otel/sdk/metric` pipeline to call the measure of specific pipeline while calling Collect method. (#5900)
```

Comment:
Describe the change from a user perspective.
```suggestion
- Fix incorrect metrics generated from callbacks when multiple readers are used in `go.opentelemetry.io/otel/sdk/metric`. (#5900)
```

Please also move this under the `Fixed` heading.

Reply:
Done.